mkdocs.yml

```diff
@@ -387,6 +387,8 @@ nav:
             - "-XX:[+|-]JITServerLogConnections"                                 : xxjitserverlogconnections.md
             - "-XX:JITServerMetrics"                                             : xxjitservermetrics.md
             - "-XX:JITServerMetricsPort"                                         : xxjitservermetricsport.md
+            - "-XX:JITServerMetricsSSLKey"                                       : xxjitservermetricssslkey.md
+            - "-XX:JITServerMetricsSSLCert"                                      : xxjitservermetricssslkey.md
```

Comment:
Do entries have to be ordered alphabetically?

Reply:
Yes, I have reordered it accordingly. Thanks!